common/djangoapps/third_party_auth/api/permissions.py

```diff
@@ -0,0 +1,29 @@
+"""
+Third party auth API related permissions
+"""
+from rest_framework import permissions
+
+from third_party_auth.models import ApiPermissions
+
+
+class ThirdPartyAuthApiPermission(permissions.BasePermission):
```

Comment:
Can we rename this to `ProviderApiPermission` since it's scoped to having access to a provider?